MaterialDesignThemes.Wpf/DataGridTextColumn.cs

```diff
@@ -15,6 +17,30 @@ public class DataGridTextColumn : System.Windows.Controls.DataGridTextColumn
             return null;
         }
 
+        protected override FrameworkElement? GenerateElement(DataGridCell cell, object dataItem)
+        {
+            var element = base.GenerateElement(cell, dataItem);
+
+            if (
+                (cell.Column as MaterialDesignThemes.Wpf.DataGridTextColumn)?.ElementStringFormat is string stringFormat
```

Comment:
Consider using pattern matching for the first part of this condition as well; effectively splitting it into 2 conditions.